app/src/main/java/com/kickstarter/features/search/ui/SearchAndFilterActivity.kt

```diff
@@ -105,6 +108,12 @@ class SearchAndFilterActivity : ComponentActivity() {
                             } else {
                                 startProjectActivity(projAndRef)
                             }
+                        },
+                        onDismissBottomSheet = {
+                            viewModel.updateParamsToSearchWith(
```

Comment:
Hardcoded values, categoryBottomSheet callback when the user has selected a category should provide the category with which to call this method. 
Same for sortingBottomSheet 

cc: @leighdouglas @ycheng-kickstarter